src/index.ts

```diff
@@ -226,8 +233,26 @@ class LiskMigrator extends Command {
 		cli.action.stop();
 
```

Comment:
Additionally, please consider wrapping the entire logic within the `run` method inside of a `try...catch` block. Within the `catch` block please log the error message and exit the application gracefully.